content/2017-munich/sponsor.md

```diff
@@ -4,22 +4,22 @@ title: Become a Sponsor
 
 ## Become a sponsor
 
-As a community conference with a minimal entrance fee, we depend on sponsors for
+As a community conference with a low entrance fee, we depend on sponsors for
 making PromCon 2017 possible.
 
 Do you want to help make this happen and **get your company featured in videos,
 banners, and printed materials?** Please let us know via the form below and we
 will be in touch with you about details:
 
-<a class="btn btn-default btn-lg" href="https://docs.google.com/forms/d/e/1FAIpQLSf2IbBDQ6qz5xRin1gb-yCLtnZMvqQthHzZT2DSlUUvOskQGg/viewform" target="_blank" role="button">
+<a class="btn btn-default btn-lg" href="TODO" target="_blank" role="button">
```

Comment:
Once we agree on conditions, I'll put a Google form with the right levels in here (and below).